Gruntfile.js

```diff
@@ -131,6 +128,45 @@ module.exports = function(grunt) {
           from: "openpgp.js",
           to: "openpgp.min.js"
         }]
+      },
+      lightweight_build: {
+        src: [
+          'dist/openpgp.js',
+          'dist/openpgp.js'
+        ],
+        overwrite: true,
+        replacements: [
+          {
+            from: "external_indutny_elliptic: false",
+            to: "external_indutny_elliptic: true"
+          }
+        ]
+      },
+      exclude_elliptic_build: {
+        src: [
+          'dist/openpgp.js',
+          'dist/openpgp.js'
+        ],
+        overwrite: true,
+        replacements: [
+          {
+            from: "use_indutny_elliptic: true",
+            to: "use_indutny_elliptic: false"
+          }
+        ]
+      },
```

Comment:
I don't think you have to create a grunt command and option for this. If they want to disable elliptic, they can do themselves:
```js
openpgp.config.use_indutny_elliptic = false;
```

Reply:
Since we have version without elliptic, let's have an option for an automatic build of this version, and then testing it.

Reply:
If you want to test it, you can set the config option in the test suite, no? We shouldn't have a separate build for every single config option, it would run out of hand.